app/src/main/java/com/example/dogbook/common/ParseApplication.java

```diff
@@ -35,12 +43,19 @@ public void onCreate() {
         );
     }
 
-    public static ParseQuery<Post> getLocationPostQuery() {
-        ParseQuery<Post> query = ParseQuery.getQuery(Post.class);
-        query.orderByDescending("createdAt");
-        query.include("author");
-        query.whereExists("location");
-        return query;
+    public static void functionTry() {
```

Comment:
Let's make this name a bit better when it comes to non-skeleton code.  What are you "trying"?